### PR TITLE
Firebird Database Support

### PR DESCRIPTION
**Pull request by andrevanzuydam**
@axelfontaine Removed my changes to the pom files and fixed the white space issue in Schema.java, that fell over when I was debugging, thanks for catching that.

### CODE REVIEW

File: flyway-core/src/main/java/org/flywaydb/core/internal/command/DbMigrate.java [hunk 1]
```diff
@@ -204,11 +204,19 @@ private Integer migrateGroup(boolean firstRun) {
             currentSchemaVersion = infoService.current().getVersion();
         }
         if (firstRun) {
-            LOG.info("Current version of schema " + schema + ": " + currentSchemaVersion);
-
-            if (configuration.isOutOfOrder()) {
-                LOG.warn("outOfOrder mode is active. Migration of schema " + schema + " may not be reproducible.");
+            if (schema.getName() != "null") {
```
Comment (seregamorph) [inline flyway-core/src/main/java/org/flywaydb/core/internal/command/DbMigrate.java]: illegal equals check for strings

Comment (lvthillo) [reply flyway-core/src/main/java/org/flywaydb/core/internal/command/DbMigrate.java]: @andrevanzuydam ?

Comment (andrevanzuydam) [reply flyway-core/src/main/java/org/flywaydb/core/internal/command/DbMigrate.java]: I'm open to suggestions, I would need to build the project again to see what the implications were, Firebird does not have schemas

**Comment by alextercete** [comment]
Superseded by #2376.

**Pull request closed without merging**